dockstore-webservice/src/main/resources/openapi3/openapi.yaml

```diff
@@ -8703,6 +8745,8 @@ components:
     VersionMetadata:
       type: object
       properties:
+        description:
+          type: string
```

Comment:
missing description for this description

Reply:
The description of this description has been described.